api/src/main/java/org/openmrs/module/ugandaemrreports/reporting/dataset/definition/SharedDataDefintion.java

```diff
@@ -36,4 +53,61 @@ public DataDefinition definition(String name, Concept concept) {
         obsForPersonDataDefinition.setWhich(TimeQualifier.LAST);
         return obsForPersonDataDefinition;
     }
+    public DataDefinition getPreARTNumber()
+    {
+        PatientIdentifierType preARTNo = MetadataUtils.existing(PatientIdentifierType.class, "e1731641-30ab-102d-86b0-7a5022ba4115");
+        DataDefinition identifierDefPre = new ConvertedPatientDataDefinition("identifier", new PatientIdentifierDataDefinition(preARTNo.getName(), preARTNo), identifierFormatter);
+        return identifierDefPre;
+    }
+    public PersonAttributeType getPatientNationality()
+    {
+        PersonAttributeType nationality = Context.getPersonService().getPersonAttributeTypeByUuid("dec484be-1c43-416a-9ad0-18bd9ef28929");
+         return nationality;
+    }
+    public PersonAttributeType getPhoneNumber()
+    {
+        PersonAttributeType phoneNumber = Context.getPersonService().getPersonAttributeTypeByUuid("14d4f066-15f5-102d-96e4-000c29c2a5d7");
+         return phoneNumber;
+    }
+    public PersonAttributeType getAlternatePhoneNumber()
```

Comment:
@solemabrothers Also add getSecondAlternatePhoneNumber()

Reply:
@ssmusoke where do you suggest we pick this Second alternate number as we have space for only two phone numbers in the data collection

Reply:
@solemabrothers look at the registration form - it has 3 phone numbers

Reply:
@ssmusoke this is a screenshot of contact information from the registration form we have only two contacts. Maybe we create a card for another phone number.
![telephone number](https://user-images.githubusercontent.com/37846623/80067723-10d83b00-8547-11ea-9a1f-8c890ec5596e.PNG)